custom_components/miele/miele_at_home.py

```
@@ -175,6 +186,29 @@ def _get_cached_token(self):
 
         return token
 
+    def _delete_token(self):
+        if self._cache_path:
+            try:
+                os.remove(self._cache_path)
+
+            except IOError:
+                pass
```

I think we should throw an error here